src/SIL.XForge.Scripture/ClientApp/e2e/src/login.po.ts

```diff
@@ -29,8 +29,14 @@ export class LoginPage {
     await this.loginButton.click();
 
     // wait for redirect before enabling angular wait
-    await browser.sleep(300);
-    browser.waitForAngularEnabled(true);
+    //await browser.sleep(300);
+    // browser.waitForAngularEnabled(true);
+    browser.wait(() => {
+      browser.waitForAngularEnabled(false);
```

Comment:
This line isn't needed; its already set to false at the beginning of this method.

Reply:
Okay, We tried to remove the "waitForAngularEnabled()" which throws the error. since async/await is continue to work from loginPage.get(). So, "waitForAngularEnabled() needs to be set false for the second time too. we could not remove that line. Thank you.